packages/portfolio-contract/src/type-guards.ts

```diff
@@ -221,6 +221,13 @@ type FlowStatus = {
   error?: string;
 };
 
+type ChainAccountStatus = {
+  nonce?: number;
+  accountId?: AccountId;
+  lca?: string;
+  status: 'Pending' | 'Created' | 'Failed';
```

Comment:
In the `Failed` case, is there some place to get an error message or something?

is this case tested?